tests/components/deconz/test_switch.py

```diff
@@ -61,34 +71,21 @@ async def test_platform_manually_configured(hass):
 
 async def test_no_switches(hass):
     """Test that no switch entities are created."""
-    gateway = await setup_deconz_integration(hass)
-    assert len(gateway.deconz_ids) == 0
+    await setup_deconz_integration(hass)
     assert len(hass.states.async_all()) == 0
 
 
-async def test_switches(hass):
+async def test_power_plugs(hass):
     """Test that all supported switch entities are created."""
     data = deepcopy(DECONZ_WEB_REQUEST)
-    data["lights"] = deepcopy(SWITCHES)
+    data["lights"] = deepcopy(POWER_PLUGS)
     gateway = await setup_deconz_integration(hass, get_state_response=data)
-    assert "switch.on_off_switch" in gateway.deconz_ids
-    assert "switch.smart_plug" in gateway.deconz_ids
-    assert "switch.warning_device" in gateway.deconz_ids
-    assert "switch.unsupported_switch" not in gateway.deconz_ids
-    assert "switch.on_off_relay" in gateway.deconz_ids
-    assert len(hass.states.async_all()) == 5
-
-    on_off_switch = hass.states.get("switch.on_off_switch")
-    assert on_off_switch.state == "on"
-
-    smart_plug = hass.states.get("switch.smart_plug")
-    assert smart_plug.state == "off"
-
-    warning_device = hass.states.get("switch.warning_device")
-    assert warning_device.state == "on"
 
-    on_off_relay = hass.states.get("switch.on_off_relay")
-    assert on_off_relay.state == "on"
+    assert len(hass.states.async_all()) == 4
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)